lib/json_spec/cucumber.rb

```diff
@@ -18,6 +18,18 @@
   end
 end
 
+Then /^the (?:JSON|json)(?: response)?(?: at "(.*)")? with UUIDs should( not)? be:$/ do |path, negative, json|
+  uuid_regexp = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/
+  time_regexp = /[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]{3}Z/
+  last_json_stripped_uuids = last_json.gsub(uuid_regexp, 'UUID').gsub(time_regexp, 'TIME')
+  expected_json_stripped_uuids = json.gsub(uuid_regexp, 'UUID').gsub(time_regexp, 'TIME')
```

Comment:
this seems like, frankly, a bad way to do this. gsubbing on an unparsed json string ... ew. can this not be accomplished by excluding particular fields, uuid and created_at or the like?